libr/asm/asm.c

```diff
@@ -195,6 +195,10 @@ R_API RAsm *r_asm_new() {
 	if (!a) {
 		return NULL;
 	}
+	a->num = r_num_new (NULL, NULL, NULL);
```

Comment:
this is initialized from RCore afaik, i think its fine to have this here, but we should cover the proper reinit from core

Reply:
ok d615cdb614476938681cc0a6fd477604d2e94f9f